TrackletAlgorithm/AllStubMemory.h

```diff
@@ -5,8 +5,6 @@
 #include "MemoryTemplate.h"
 #include "globalFunctions.h"
 
-// Memory containing full precision stubs written by the VMRouter.
-
```

Comment:
@aryd Please restore this comment that was added in #316.

Reply:
Fixed.